analyzer/src/main/kotlin/AnalyzerResultBuilder.kt

```diff
@@ -43,15 +44,15 @@ class AnalyzerResultBuilder {
     private val issues = mutableMapOf<Identifier, List<Issue>>()
     private val dependencyGraphs = mutableMapOf<String, DependencyGraph>()
 
-    fun build(): AnalyzerResult {
+    fun build(excludes: Excludes = Excludes()): AnalyzerResult {
```

Comment:
Should this re-use `PackageManager.EMPTY_EXCLUDES` (which would need to become at least `internal` then)?

Reply:
Now using the new `Excludes.EMPTY` constant.